tools/trim_adapters/trim_adapters.xml

```diff
@@ -0,0 +1,71 @@
+<tool id="trim_adapters" name="Trim putative adapter sequence" version="0.1.0">
+    <requirements>
+        <requirement type="package" version="0.1.6">atactk</requirement>
+    </requirements>
+    <command detect_errors="exit_code"><![CDATA[
+        ln -s '$input1' ./forward.fastq &&
```

Comment:
This will not work if you include a fastq.gz file as the ending needs to end with gz I assume. You could add an additional test for it and this line needs to become: `./forward.${input1}.ext`, same for $input2